package.json

```diff
@@ -88,7 +90,7 @@
     "ethers": "6.14.4",
     "glob": "11.0.3",
     "globals": "15.15.0",
-    "hardhat": "2.26.0",
+    "hardhat": "2.26.1",
```

Comment:
ℹ️ We run on 2.26.0 in CI.

Reply:
I see not problem to have hardhat-node on another "patch" version - it's just tedious to upgrade hardhat-node repo each time